client/src/api/meals.js

```diff
@@ -1,6 +1,10 @@
-export const getMeals = (id, query) => {
+import getTime from 'date-fns/getTime'
+
+export const getMeals = (id, { dateFrom, dateTo, asc, desc }) => {
   return fetch(
-    `${process.env.REACT_APP_API_BASE_URL}/meals/?userId=${id}&dateFrom=${query.dateFrom.date}&dateTo=${query.dateTo.date}`,
+    `${process.env.REACT_APP_API_BASE_URL}/meals/?userId=${id}&dateFrom=${
+      dateFrom ? getTime(dateFrom) : ''
+    }&dateTo=${dateTo ? getTime(dateTo) : ''}&asc=${asc}&desc=${desc}`,
```

Comment:
ეს ფუნქცია getTime-ზე დამოკიდებული არ უნდა იყოს. მას უკვე getTime-იანი dateFrom/to უნდა ჰქონდეს გადმოცემული.

Concern Separation...

ასევე, ტესტირების დროს mocking საჭირო არ იქნება